src/app/(routes)/createcard/page.tsx

```
@@ -0,0 +1,242 @@
+
+  return (
+    <div className="flex items-center justify-center min-h-screen bg-black p-[15px] lg:p-[40px] md:p-[20px]">
+      <div className="w-full max-w-[1480px]">
+        <header className="text-center md:text-left mb-[20px]">
+          <div onClick={() => router.push("/mygallery")} className="cursor-pointer block md:hidden">
+            <Image src="/icons/direction/back.svg" width={22} height={22} alt="Back Icon" />
+          </div>
+          <Title
+            variant="primary"
+            font="baskin"
+            children="포토카드 생성"
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)